mobile-app/app/screens/AppNavigator/screens/Auctions/screens/PlaceBidScreen.tsx

```diff
@@ -135,130 +135,233 @@ export function PlaceBidScreen(props: Props): JSX.Element {
-            inlineText={{
-              type: "error",
-              text:
-                bidAmount === "" || (hasSufficientFunds && isValidMinBid)
-                  ? undefined
-                  : translate(
-                      "screens/PlaceBidScreen",
-                      !hasSufficientFunds
-                        ? "Insufficient funds"
-                        : "Bid amount is lower than required"
-                    ),
-            }}
           >
-            <SetAmountButton
-              amount={new BigNumber(minNextBidInToken)}
-              onPress={onBidMinAmount}
-              type={AmountButtonTypes.max}
-              customText={translate("screens/PlaceBidScreen", "MIN. BID")}
+            <View style={tailwind("w-6/12 mr-2")}>
+              <Controller
+                control={control}
+                defaultValue=""
+                name="bidAmount"
+                render={({ field: { onChange, value } }) => (
+                  <ThemedTextInputV2
+                    style={tailwind("text-xl font-semibold-v2 w-full")}
+                    light={tailwind("text-mono-light-v2-900")}
+                    dark={tailwind("text-mono-dark-v2-900")}
+                    keyboardType="numeric"
+                    value={value}
+                    onBlur={async () => {
+                      await onChange(value?.trim());
+                    }}
+                    onChangeText={async (amount) => {
+                      amount = isNaN(+amount) ? "0" : amount;
+                      setValue("bidAmount", amount);
+                      await trigger("bidAmount");
+                    }}
+                    placeholder="0.00"
+                    placeholderTextColor={getColor(
+                      isLight
+                        ? bidAmount === ""
+                          ? "mono-light-v2-500"
+                          : "mono-light-v2-900"
+                        : bidAmount === ""
+                        ? "mono-dark-v2-500"
+                        : "mono-dark-v2-900"
+                    )}
+                    testID="text_input_bid_amount"
+                    editable={bidAmount !== undefined}
+                  />
+                )}
+                rules={{
+                  required: true,
+                  pattern: /^\d*\.?\d*$/,
+                  validate: {
+                    min: (value) => new BigNumber(value).gte(minNextBidInToken),
+                    hasSufficientFunds: (value) =>
+                      new BigNumber(ownedTokenAmount).gte(value),
+                  },
+                }}
+              />
+              <ActiveUSDValueV2
+                price={getAmountInUSDValue(bidAmount)}
+                testId="bid_value_in_usd"
+                containerStyle={tailwind("w-full break-words")}
+              />
+            </View>
+            <TokenDropdownButton
+              symbol={batch.loan.displaySymbol}
+              testID="place_bid"
```

Comment:
let's use a diff `testID` to note create confusion

Reply:
Updated